frontend/components/ui/blocknote/blocks/code-block.tsx

```
@@ -0,0 +1,520 @@
+      success: true,
+      output: output.trim(),
+    };
+  } catch (error: any) {
+    return {
+      success: false,
+      output: `Execution error: ${error.message}`,
+    };
+  }
+};
+
+const CodeBlockComponent = ({ block, editor }: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)